loom/script/native/lsLuaBridge.h

```diff
@@ -4080,6 +4084,10 @@ class Namespace
             lua_pushcclosure(L, &StaticConstructorProxy<T, FP>::call, 2);
             rawsetfield(L, -2, "__call");
 
+            lua_pushcfunction(L, &gcMetaMethod);
```

Comment:
Or maybe it was expected that the instances get cleared manually somehow? It definitely did seem missing though...